en/advanced-topics/meta.md

```diff
@@ -56,86 +56,86 @@ Cocos Creator 会为 assets 目录下的每一个文件和目录生成一个同
 }
 ```
 
-png 图片中的 meta 文件信息比较多，除了基本的 ver 和 uuid 外，还记录了图片的宽高、偏移、九宫格等数据。上面这么多信息，我们这里只关心一个：**uuid**。
+The png image has more information in the **meta file**. In addition to the basic ver and uuid, it also records the width, height, offset, and nine-square grid of the image. There is a lot of information that is stored. **uuid** is particularly important.
 
-> uuid : 通用唯一标识符（Universally Unique Identifier）
+> uuid : Universally Unique Identifier
 
-uuid 是 Creator 用来管理游戏资源的。它会为每个文件分配一个唯一的 id，图集会生成多个。由此可以了解在 Creator 引擎中，识别一个文件不是简单地通过 `路径 + 文件名` 定位，而是通过 uuid 来引用文件。因此可以在编辑器资源管理中，随意删除、移动文件。
+**uuid's** in __Cocos Creator__ are used to manage the resources of the game. It assigns a unique id to each file. This means that in the __Cocos Creator__ engine, identifying a file is not simply by `path + filename`, but by **uuid**. Therefore, you can delete and move files at will in **Editor Resource Management**.
 
-## meta 文件更新时机
+## Meta File Updating
 
-Creator 生成 meta 文件有以下几种情况：
+__Cocos Creator__ generates **meta files** the following situations:
 
-**1、打开工程时**
+  **1. when opening the project**
 
-打开项目工程时，Creator 会先扫描 assets 目录，如果哪个文件还没有 meta 文件，此时就会生成。
+  When you open a project, __Cocos Creator__, scans the `assets` directory first, and if it does not have a **meta file**, one will be generated.
 
-**2、更新资源时**
+  **2. when updating resources**
 
-更新资源也会引发 meta 文件的更新：
+  Updating resources also triggers updates to the corresponding **meta files**:
 
-- 在 **资源管理器** 中可以对资源进行文件名修改、改变目录、删除文件，添加文件等操作，请参考 [资源管理器](../getting-started/basics/editor-panels/assets.md)。也可以直接从桌面或操作系统的文件管理器中将文件拖入到 **资源管理器** 中。
+  - In **Explorer**, you can modify the file name, change the directory, delete files, add files, etc., please refer to [Explorer](../getting-started/basics/editor-panels/assets.md). You can also drag files directly into the **Explorer** from your desktop or other folder.
 
-![](meta/add.png)
+  ![](meta/add.png)
 
-- 还有一种情况是在操作系统的文件管理器中对 assets 目录中的文件进行增、删、改之后切换到编辑器界面，此时可以看到 **资源管理器** 刷新的过程。
+  - There is also a case where the files in the `assets` directory are added, deleted, changed, in the **Editor Interface**. Changes here are reflected on your local **file-system**! No need to make these changes twice, once in __Cocos Creator__ and then again on your **file-system**. The **Resource Manager** refreshes with each change.
```

Comment:
这段话的意思是：在文件系统当中修改，然后同步到编辑器。而不是在编辑器中修改然后同步到你的文件系统，这个在第一种情况里面已经介绍了。
所以我改成了下面的翻译：
In another case, in the file manager of the operating system, the files in the assets directory are added, deleted, changed, and then switched to the editor interface. At this point, the process of refreshing the **Assets** can be seen.

Reply:
Except we dont talk like that. If I read it outloud it isn't what we would say.

We dont use: 

* __in another case__ --> maybe __another example__

* __file manager of the operating system__, what do you mean here? What if someone is new and they are looking for an app called file manager? maybe they dont realize that the file manager is just clicking on folders on their desktop or where ever they store their work. Perhaps this is a bit pedantic because we are talking about developers and we assume these developers have a higher knowledge than normal every day users.

* __At this point, the process of refreshing the Assets can be seen.__ We dont talk like this either.

Reply:
那可以按照你说的方式调整一下措辞，只要意思符合原文的意思就行。我只是想跟你解释一下原文的意思。